src/desktop/components/search_bar/view.coffee

```diff
@@ -64,11 +73,38 @@ module.exports = class SearchBarView extends Backbone.View
 
   checkSubmission: (e) ->
     @hideSuggestions()
+
     return if !(e.which is 13) or @selected?
 
     unless @isEmpty()
       @trigger 'search:entered', encodeURIComponent(@$input.val())
 
+  maybeHideSpotlight: (e) ->
+    @hideSpotlight() unless e.which is 37 or e.which is 39
```

Comment:
Maybe add a comment like `# Enter / Down` so that the keycodes can be understood at a glance?

Reply:
yea i'll go in and add a few comments